AutonomousDemo/SimulationLoop.cs

```diff
@@ -10,20 +10,22 @@ namespace AutonomousDemo
     public class SimulationLoop
     {
 
-        public void Run(Vehicle vehicle)
+        public void Run(Vehicle vehicle, double distance)
         {
             while (true)
             {
-                ProcessInput(vehicle);
+                ProcessInput(vehicle, distance);
+                Thread.Sleep(1000); //just here so I can see the output
```

Comment:
I take it this is a console app and it's vanishing once you finish? You can change that by adding a Console.ReadLine() to your main() routine, so it won't go away until you hit "ENTER".

I also take it that means your simulation is running and giving you some kind of relatively sensible output. Awesome!